### DISCUSSION TITLE
Updating "Homepage" not working out of the box

### DISCUSSION THREAD

**Discussion by Scorpoon** [tteck/Proxmox#2828] [General]
I installed a fresh container with the tteck helper script and tried then to update with the update command.
As you can see in this discussion on the [homepage git](https://github.com/gethomepage/homepage/discussions/3335) i had to manually remove the node_modules folder and the package-lock.json and reinstall the dependencies, shouldn't that be a step in the install/update script?

**Comment by XxxSpiritvsxxX** [comment]
Same issue for me.
Updating Homepage breaks the container.


> Build error occurred
Error: Failed to collect page data for /browserconfig.xml
    at /opt/homepage/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0__react@18.2.0/node_modules/next/dist/build/utils.js:916:15
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  type: 'Error'
}
info  - Collecting page data . ELIFECYCLE  Command failed with exit code 1.

[ERROR] in line 75: exit code 0: while executing command pnpm build

**Comment by tteck** [comment]
@Scorpoon PRs are welcomed 😉

**Comment by Scorpoon** [comment]
I will have a look into it, good exercise to get more familiar with the shell i guess :D

**Comment by Scorpoon** [comment]
Seems to be fixed since your last commits but found another "issue" -> made an issue/PR for it
